spec/services/sign_in/token_serializer_spec.rb

```diff
@@ -128,6 +128,21 @@
           expect(subject).to eq({})
         end
       end
+
+      context 'when client_config has refresh_token_path' do
+        it 'returns the refresh_token_path from client_config' do
+          subject
+          expect(cookies[refresh_token_cookie_name][:path]).to eq(refresh_path)
+        end
+      end
+
+      context 'when client_config does not have refresh_token_path' do
+        it 'returns nil when refresh_token is not present' do
+          subject
+          cookies[refresh_token_cookie_name] = { path: nil }
+          expect(cookies[refresh_token_cookie_name][:path]).to eq(nil)
+        end
+      end
```

Comment:
This is not what is expected. When the `refresh_token_path` is `nil` it should return the default.